app/services/rapid_ready_for_decision/constants.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module RapidReadyForDecision
+  class Constants
+    DISABILITIES = {
+      hypertension: {
+        code: 7101,
+        label: 'hypertension',
+        flipper_name: 'hypertension'
+      },
+      asthma: {
+        code: 6602,
+        label: 'asthma',
+        flipper_name: 'asthma'
+      }
+    }.freeze
+
+    DISABILITIES_BY_CODE = DISABILITIES.map { |_k, v| [v[:code], v] }.to_h
```

Comment:
@allthesignals This is what I ended up doing to rekey the `DISABILITIES` hash by diagnostic code.